code/default/launcher/win_tray.py

```diff
@@ -12,28 +12,28 @@
     sys.path.append(win32_lib)
 
 import webbrowser
-from systray import SysTrayIcon
-import systray.win32_adapter as win32_adapter
 import os
 import ctypes
 import _winreg as winreg
 import win32_proxy_manager
-
 import module_init
 import update
-from instances import xlog
 import config
 
+from instances import xlog
+from systray import SysTrayIcon, win32_adapter
+
+import locale
+lang_code, code_page = locale.getdefaultlocale()
 
 class Win_tray():
     def __init__(self):
         icon_path = os.path.join(os.path.dirname(__file__), "web_ui", "favicon.ico")
-        self.systray = SysTrayIcon(icon_path, "XX-Net", self.make_menu(), self.on_quit, left_click=self.on_show, right_click=self.on_right_click)
+        self.systray = SysTrayIcon(icon_path, "XX-Net", 
+            self.make_menu(), self.on_quit, left_click=self.on_show, right_click=self.on_right_click)
```

Comment:
妹纸，俺爱你就像老鼠爱大米……
Cat73 <notifications@github.com>于2016年6月18日 周六09:00写道：

> In code/default/launcher/win_tray.py
> <https://github.com/XX-net/XX-Net/pull/3668#discussion_r67592117>:
>
> >
> >  class Win_tray():
> >      def __init__(self):
> >          icon_path = os.path.join(os.path.dirname(__file__), "web_ui", "favicon.ico")
> > -        self.systray = SysTrayIcon(icon_path, "XX-Net", self.make_menu(), self.on_quit, left_click=self.on_show, right_click=self.on_right_click)
> > +        self.systray = SysTrayIcon(icon_path, "XX-Net",
> > +            self.make_menu(), self.on_quit, left_click=self.on_show, right_click=self.on_right_click)
>
> 好吧看错了，这里不需要缩进- -
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/XX-net/XX-Net/pull/3668/files/153605f7becaa4159852f9f230ad22e81b3c6c1b#r67592117>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMtfyBqzWvpAV7Z1utmHkbBSnsvlWNcMks5qM0M3gaJpZM4I4W2D>
> .
>